README.md

```diff
@@ -33,6 +33,8 @@ composer require yiisoft/files --prefer-dist
 
 FileHelper provides static methods you can use for various filesystem-related operations.
 
+Pattern matching is based on `yiisoft/strings` package. See more information here https://github.com/yiisoft/strings#wildcardpattern-usage
```

Comment:
Seems, need create new section about patch matchers to which to add info about `PathMatcher` and `CompositePathMatcher` and add this link also.